eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -94,6 +94,29 @@ jobs:
   - checkout: self
     submodules: recursive
 
+  # Validate all the files are formatted correctly according to the .clang-format file.
+  - bash: |
+      # Run clang-format recursively on each source and header file within the repo sdk folder.
+      echo Check clang-formatting
+      clang-format --version
+      find sdk/ -regex '.*\.\(cpp\|hpp\)' -exec clang-format -style=file -i {} \;
```

Comment:
Do we need the command option `-style=file`?

> When using -style=file, clang-format for each input file will try to find the .clang-format file located in the closest parent directory of the input file. When the standard input is used, the search is started from the current directory.

We have a global .clang-format file.

Reply:
I am not sure if by not adding it, clang-format will try to use the .clang-format file or not

Reply:
> I am not sure if by not adding it, clang-format will try to use the .clang-format file or not

You can test that by changing the clang-format file and see if re-formatting the files causes a change. It works in C.

Reply:
I just checked and it is the default option. So, it is actually the same. 
I think it is good to have so it is clear

Reply:
> I just checked and it is the default option. So, it is actually the same.

Can you share a source which mentions that's the default value of `style` when that option isn't specified?